avoids:
ld.lld: error: version script assignment of 'local' to symbol '_fini' failed: symbol not defined
ld.lld: error: version script assignment of 'local' to symbol '_init' failed: symbol not defined

--- a/make/data/hotspot-symbols/version-script-clang.txt
+++ b/make/data/hotspot-symbols/version-script-clang.txt
@@ -1,8 +1,4 @@
 SUNWprivate_1.1 {
   global:
     *;
-
-  local:
-    _fini;
-    _init;
 };
